packages/wrangler/src/deployment-bundle/find-additional-modules.ts

```diff
@@ -14,18 +15,23 @@ import type { ParsedRules } from "./rules";
 import type { CfModule } from "./worker";
 
 async function* getFiles(
-	root: string,
-	relativeTo: string
+	configPath: string | undefined,
+	moduleRoot: string,
+	relativeTo: string,
+	projectRoot: string
 ): AsyncGenerator<string> {
-	for (const file of await readdir(root, { withFileTypes: true })) {
+	const wranglerHiddenDirPath = getWranglerHiddenDirPath(projectRoot);
+	for (const file of await readdir(moduleRoot, { withFileTypes: true })) {
+		const absPath = path.join(moduleRoot, file.name);
 		if (file.isDirectory()) {
-			yield* getFiles(path.join(root, file.name), relativeTo);
+			// Skip the hidden Wrangler directory and Wrangler config path, so we don't accidentally bundle non-user files.
+			if (absPath !== wranglerHiddenDirPath && absPath !== configPath) {
```

Comment:
This seems to only be checking for the Wrangler config path inside an `isDirectory()` check, which I think will always be false?

Reply:
Good spot!

Reply:
36848452fc8bf1cce5352ea3c73fc3c5c3a95ef5